bin/jest-it-up

```diff
@@ -16,6 +16,10 @@ program
   .option('-i, --interactive', 'ask for confirmation before applying changes')
   .option('-s, --silent', 'do not output messages')
   .option('-d, --dry-run', 'process but do not change files')
+  .option(
+    '-w, --whole-number',
+    'round coverage thresholds down to whole a whole number',
+  )
```

Comment:
This is great, but I think we could have a more generic solution:

```suggestion
  .option('-p, --precision <digits>',
    'number of threshold decimal places to persist',
    parseInt,
    2,
  )
```

You can then do `+desiredCoverage.toFixed(precision)` in `getNewThresholds`, and pass `--precision 0` if you want whole numbers. What do you think?

Reply:
I like the idea of what you have proposed however i just found out that if the coverage is 30.6, and we want precision 0 it will then round up to 31. technically the coverage isnt at 31 so would fail i think

Reply:
Oh right, I forgot `toFixed()` rounds the number. In that case we would need something like this:

```js
const factor = Math.pow(10, precision)
Math.trunc(desiredCoverage * factor) / factor
```

It may still round numbers due to floating point arithmetic though (e.g. `4.27` with precision `2` → `4.26`), but I think that's acceptable, especially since it's opt-in. Alternatively, we could manipulate the number as a string, but that can get ugly pretty quickly.